content/blog/2021/10/05/OptimizeYourPython.adoc

```diff
@@ -0,0 +1,438 @@
+= Optimize your Python
+Christopher-Chianelli
+2021-10-05
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: python, integration
```

Comment:
Can we add a :jbake-social_media_share_image: ?

How about something like [optaplanner icon + python icon = "OptaPy"]?
We can add it under the first paragraph too. I 'll quickly create it and mail it to you - your call if you want to add it.